starfyre/__main__.py

```diff
@@ -13,42 +15,27 @@ def write_js_file(path):
     dist_path.mkdir(exist_ok=True)
     js_store = pkg_resources.path("starfyre.js", "store.js")
     shutil.copy(str(js_store), path + "/dist/store.js")
-
-
+    
 def create_main_file(path):
     """
     Creates a main file in the build directory.
     
     This file will be used to run the project.
-    Every starfyre build will have an `__init__.py` file in the build directory.
+     Every starfyre build will have an `__init__.py` file in the build directory.
     And the `__init__.py` file will have a component that will render the root component. It will be named `app`.
 
     You can have a look at the `test-application/build/__init__.py` file to see what it looks like.
 
-    The main file is also responsible for adding the `store.js` file to the `index.html` file.
+    The main file is also responsible for adding the `store.js` file
     """
     output_file_path = path + "/build/__main__.py"
     write_js_file(path)
 
     with open(output_file_path, "w") as f:
         f.write(
-            """
-from . import app
-import os
-from pathlib import Path
-
-
-if __name__ == '__main__':
-    path_ = os.path.dirname(os.path.abspath(__file__))
-    directory = Path(path_ ) / ".." / "dist"
-    if not directory.exists():
-        directory.mkdir()
-
-    with open(f"{directory}/index.html", "w") as f:
-        f.write("<script src='store.js'></script>")
-        f.write(app)
-"""
-        )
+"""if __name__ == '__main__':
+    pass
+    """)
```

Comment:
I removed the route generation from this function. This function will be used to call `write_to_js` function and also used to create `build/__main.py__` file which will serve as the entry point for the application.